FF1Lib/FF1Rom.cs

```diff
@@ -167,9 +167,10 @@ public void Randomize(Blob seed, Flags flags, Preferences preferences)
 			}
 #endif
 
-			if (flags.RandomizeFormationEnemizer)
+			if (flags.RandomizeEnemizer || flags.RandomizeFormationEnemizer)
 			{
-				DoEnemizer(rng, false, flags.RandomizeFormationEnemizer, false);
+				FixEnemyPalettes(); // fixes a big in the original game's programming that causes third enemy slot's palette to render incorrectly

```

Comment:
`FixEnemyPalettes()` is totally harmless right? It's a pure fix? You can just call it at the top of Randomize() like with do with a bunch of other things.

Reply:
Yes, it just removes an LSR A in the middle of the game's programming.  The disassembly notes it as a bug.